roles/de-deploy-service/tasks/restart-service.yaml

```diff
@@ -1,49 +1,43 @@
-# Performs a basic deployment of a service: pulls it and dependent images like configs and iplant_data, stops and removes existing containers, then starts the service.
+# Performs a basic deployment of a service: pulls it and dependent images like configs and the data container,
+# stops and removes existing containers, then starts the service.
 ---
-- block:
```

Comment:
I pulled these tasks out of the block because the environment variables are now being set in higher-level task files, which obviates the need for the block.